README.md

```diff
@@ -26,23 +28,32 @@ The main components of the crate are:
 
 This allocator should be used to allocate resources that can be reduced to an
 integer type like legacy GSI numbers or KVM memory slot IDs. The
-characteristics of such a resource are represented by the `IdAllocator` struct.
-
-The struct that defines the `IdAllocator` contains the end of the interval that
-is managed, a field that points at the next available ID and a `BTreeSet` that
-is used to store the released IDs. The reason for using a `BTreeSet` is that
-the average complexity for deletion and insertion is `O(log N)`, offering a
+characteristics of such a resource are represented by the 
+[`IdAllocator`](https://docs.rs/vm-allocator/latest/vm_allocator/struct.IdAllocator.html) 
```

Comment:
Hmm, do we actually want to link these things every time? I think it would be enough to do that for only the first occurrence.

Reply:
No strong preference on my side.